pkg/network/tracer/testutil/kafka/testdata/docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: '3'
```

Comment:
Consider adding a reference to the original file/blog post (zk-single-kafka-single.yml)

Reply:
Copied it from your branch, can you share the origin?

Reply:
https://github.com/conduktor/kafka-stack-docker-compose/blob/master/zk-single-kafka-single.yml

Reply:
The blog: https://www.conduktor.io/kafka/how-to-start-kafka-using-docker